### PR TITLE
adds styling + Drafts to My Pipelines list

### PR DESCRIPTION
**Pull request by astewes**
*  Adds styling to My Pipelines
*  Removes drafts state and adds drafts to My Pipelines
*  Removes related styling from etlapps.less that will no longer be used
*  Fix datasets view in Apps Detail for ETL Apps
*  Finish List view for Hydrator
*  Add confirmation for delete pipeline operation (both in list & detail view)
*  Status is now sortable

TO DO: 
- [x] Fix drafts functionality inside list-ctrl.js
- [x] Add drafts to ng-repeat in table
- [ ] Get top panel status icons (per the mock) from Sean
- [x] Add all remaining integrations to top panel and table

mp

### CODE REVIEW

File: cdap-ui/app/features/adapters/controllers/list-ctrl.js [hunk 1]
```diff
@@ -28,5 +30,60 @@ angular.module(PKG.name + '.feature.adapters')
       .$promise
       .then(function success(res) {
         this.adaptersList = res;
+        console.log(this.adaptersList);
```
Comment (anwar6953) [inline cdap-ui/app/features/adapters/controllers/list-ctrl.js]: remove?

File: cdap-ui/app/features/adapters/controllers/list-ctrl.js [hunk 2]
```diff
@@ -28,5 +30,60 @@ angular.module(PKG.name + '.feature.adapters')
...
       }.bind(this));
+
+      mySettings.get('adapterDrafts')
+        .then(function(res) {
+          if (res && Object.keys(res).length) {
+            angular.forEach(res, function(value, key) {
+              this.drafts.push({
+                isdraft: true,
```
Comment (anwar6953) [inline cdap-ui/app/features/adapters/controllers/list-ctrl.js]: Is this field unused?

File: cdap-ui/app/features/adapters/controllers/list-ctrl.js [hunk 3]
```diff
@@ -28,5 +30,60 @@ angular.module(PKG.name + '.feature.adapters')
...
+            if (res[draftName]) {
+              delete res[draftName];
+            }
+            return mySettings.set('adapterDrafts', res);
+          })
+          .then(
+            function success() {
+              var alertObj = {
+                type: 'success',
+                content: 'Adapter draft ' + draftName + ' delete successfully'
```
Comment (anwar6953) [inline cdap-ui/app/features/adapters/controllers/list-ctrl.js]: I think it should either be:
"deleted succesfully"
or
"delete successful"

**Comment by elfenheart** [comment]
new screenshot:
screen shot 2015-09-16 at 3 49 46 pm

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**